blackduck-security-task/src/blackduck-security-task/bridge.ts

```diff
@@ -57,34 +57,56 @@ import { ErrorCode } from "./enum/ErrorCodes";
 
 export class Bridge {
   bridgeExecutablePath: string;
+  bridgeVersion: string;
   bridgeArtifactoryURL: string;
   bridgeUrlPattern: string;
   bridgeUrlLatestPattern: string;
 
   constructor() {
     this.bridgeExecutablePath = "";
+    this.bridgeVersion = "";
     this.bridgeArtifactoryURL =
-      "https://repo.blackduck.com/bds-integrations-release/com/blackduck/integration/bridge-cli";
+      "https://artifactory.internal.synopsys.com/artifactory/clops-local/clops.sig.synopsys.com/bridge/binaries/bridge-cli-bundle";
     this.bridgeUrlPattern = this.bridgeArtifactoryURL.concat(
-      "/$version/bridge-cli-$version-$platform.zip"
+      "/$version/bridge-cli-bundle-$version-$platform.zip"
     );
     this.bridgeUrlLatestPattern = this.bridgeArtifactoryURL.concat(
-      "/latest/bridge-cli-$platform.zip"
+      "/latest/bridge-cli-bundle-$platform.zip"
     );
   }
 
   async extractBridge(fileInfo: DownloadFileResponse): Promise<string> {
-    const extractZippedFilePath: string =
+    const bridgeInstallDirectory: string =
       inputs.BRIDGECLI_INSTALL_DIRECTORY_KEY || this.getBridgeDefaultPath();
 
+    const bridgeCliFullPath = path.join(
+      String(bridgeInstallDirectory),
+      String(this.getDefaultBridgeSubDirectory())
+    );
+    taskLib.debug("bridgeCliFullPath: " + bridgeCliFullPath);
+
     // Clear the existing bridge, if available
-    if (taskLib.exist(extractZippedFilePath)) {
-      await taskLib.rmRF(extractZippedFilePath);
+    if (taskLib.exist(bridgeCliFullPath)) {
+      await taskLib.rmRF(bridgeCliFullPath);
     }
 
-    await extractZipped(fileInfo.filePath, extractZippedFilePath);
+    await extractZipped(fileInfo.filePath, bridgeInstallDirectory);
 
-    return Promise.resolve(extractZippedFilePath);
+    if (this.bridgeVersion != "") {
+      const bridgePathWithVersion = path.join(
```

Comment:
use bridgeCli for all variables instead of bridge to keep it consistent with other parameters.

Reply:
update bridge to bridgeCli everywhere